Jetsurvey/settings.gradle

```diff
@@ -1,2 +1,17 @@
-include ':app'
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+include(":app")
 rootProject.name = "Jetsurvey"
+enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
Version catalogs are stable since Gradle 7.4, so they should work without this line